README.md

```diff
@@ -72,6 +72,16 @@ See below for how to configure the environment variables appropriately
 - Awattar__VATMultiplier=1.00 # Optional (default: 1.19), you should not need to set this unless your VAT differs from the default
 ```
 
+### Barry
+
+```yaml
+- TeslaMate__EnergyProvider=Barry
+- Awattar__VATMultiplier=1.00 # Optional (default: 1.19), you should not need to set this unless your VAT differs from the default
```

Comment:
I assume this line can be removed? I can't see any VAT Multiplier implementation for Barry